components/webcompat/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+extensions/common",
```

Comment:
not sure but probably you need `net` and `url`

Reply:
Removed this file as it is no longer needed.